frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -83,7 +83,7 @@
             <slot name="table-title">
               <th class="text-center" scope="col">{{ T.wordsProblem }}</th>
             </slot>
-            <th class="text-center" scope="col">{{ T.wordsAuthor }}</th>
+            <th class="text-center" scope="col">{{ T.tableHeaderAskedBy }}</th>
```

Comment:
aquí el hecho de que es una cabecera no aporta mucho al nombre de la cadena: si la quisiéramos poner en otro contexto, se vería raro si no es una cabecera. en este caso sería mejor indicar que es algo que pertenece a clarificaciones:

```suggestion
            <th class="text-center" scope="col">{{ T.clarificationsAskedBy }}</th>
```